README.md

```diff
@@ -15,10 +15,10 @@ Other IDEs like IntelliJ IDEA, Eclipse, Visual Studio, and CLion are also suppor
 ## Getting Started - Creating a new project
 ### With Visual Studio Code (recommended)
 For getting started with VS Code, please see the frc-docs documentation:
-https://docs.wpilib.org/en/latest/docs/software/getting-started/index.html
+https://docs.wpilib.org/en/latest/docs/zero-to-robot/step-3/creating-benchtop-test-program-cpp-java.html
```

Comment:
I'd rather link to https://docs.wpilib.org/en/latest/docs/zero-to-robot/introduction.html as the benchtop article I plan to phase out.